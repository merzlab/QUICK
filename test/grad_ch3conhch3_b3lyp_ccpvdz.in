DFT B3LYP BASIS=CC-PVDZ SCF=200 CHARGE=0 MULT=1 GRADIENT DIPOLE
 
 H         5.4269280037321330       5.8668001759742197       1.9440424210060896 
 C         5.2778863555062401       6.1653696715003079       0.8517883878991734 
 H         5.9665759620204497       6.8917325525739459       0.4522474046750474 
 H         4.4104954720977245       6.6466044949751728       0.9546987362515305 
 C         5.1169729983679595       4.9326993945007960      -0.0230474591437522 
 O         4.1460063240795630       4.6757957711199207      -0.6787269435924508 
 N         6.1595314449254053       4.0934234816143507       0.1359075710503025 
 H         6.7649147303327437       4.3215394902951143       0.9156516725231513 
 C         6.2481130228304949       2.8038163311913320      -0.5367658543372317 
 H         5.5175447980454653       2.6865231194871195      -1.3190682664333815 
 H         7.1867685623498705       2.6684759996505929      -1.0046972369643465 
 H         6.1319761352814020       2.0088922338612880       0.2503952716044978

#ref_grad    -0.0151780998
#ref_grad    -0.0046526290
#ref_grad     0.0224292967
#ref_grad    -0.0428955867
#ref_grad     0.0417630231
#ref_grad    -0.0236136475
#ref_grad    -0.0190626664
#ref_grad    -0.0049163030
#ref_grad    -0.0008238439
#ref_grad     0.0829144101
#ref_grad    -0.0415336146
#ref_grad     0.0032583186
#ref_grad    -0.0097402838
#ref_grad    -0.0089230282
#ref_grad    -0.0315679396
#ref_grad     0.0284187991
#ref_grad     0.0006693281
#ref_grad     0.0277730524
#ref_grad    -0.0133708417
#ref_grad     0.0096516632
#ref_grad     0.0111881918
#ref_grad    -0.0060747927
#ref_grad     0.0063966308
#ref_grad     0.0047937542
#ref_grad     0.0088141899
#ref_grad     0.0063155498
#ref_grad    -0.0416764549
#ref_grad     0.0168846846
#ref_grad    -0.0044866386
#ref_grad     0.0030195104
#ref_grad    -0.0319900718
#ref_grad     0.0021076096
#ref_grad     0.0106351532
#ref_grad     0.0012879197
#ref_grad    -0.0025233011
#ref_grad     0.0147043062
#TOTAL_ENERGY= -248.517841409

