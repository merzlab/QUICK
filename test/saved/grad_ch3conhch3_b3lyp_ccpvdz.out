  *********************************************************************************************************
  **                                                                                                     **
  **           888888888888                                                                              **
  **         8888888888888888                                                                            **
  **      888888888888888888                                                                             **
  **     888888888888888888                                                                              **
  **    888888888888888                                                                                  **
  **   88888888888888888888                               88888                       8888:              **
  **   8888888888888888888888Z                            88888                       8888:              **
  **   888888888888888888888888?                          88888                       8888:              **
  **   8888888888888      8888888                                        888888       8888:              **
  **    88888888888         888888     8888:     88888    88888        888888888I     8888:    888888    **
  **    8888888888           88888:    8888:     88888    88888      $888888888888    8888:   888888     **
  **    I8Z 88888             88888    8888:     88888    88888    .888888     8888   8888: 888888       **
  **    .8Z 88888             88888    8888:     88888    88888    $88888             8888:88888         **
  **     8I 88888      .=88. .88888    8888:     88888    88888    88888              8888888888         **
  **    :8  88888      888888$8888$    8888:     88888    88888    8888O              88888888888        **
  **   ,7   +88888.     8888888888.    8888:     88888    88888    888888             88888O888888       **
  **         $888888:.   .8888888      88888....888888    88888     888888     8888   8888:  888888      **
  **          I8888888888888888888     888888888888888    88888     O888888888888O    8888:   888888     **
  **            O888888888888888888     88888888888888    88888       88888888888$    8888:    888888    **
  **               8888888Z     888      .8888I  88888    88888         8888888       8888:     888888   **
  **                                                                                                     **
  **                                                                                                     **
  **                                         Copyright (c) 2021                                          **
  **                          Regents of the University of California San Diego                          **
  **                                    & Michigan State University                                      **
  **                                        All Rights Reserved.                                         **
  **                                                                                                     **
  **                   This software provided pursuant to a license agreement containing                 **
  **                   restrictions on its disclosure, duplication, and use. This software               **
  **                   contains confidential and proprietary information, and may not be                 **
  **                   extracted or distributed, in whole or in part, for any purpose                    **
  **                   whatsoever, without the express written permission of the authors.                **
  **                   This notice, and the associated author list, must be attached to                  **
  **                   all copies, or extracts, of this software. Any additional                         **
  **                   restrictions set forth in the license agreement also apply to this                **
  **                   software.                                                                         **
  *********************************************************************************************************
                                                                                                           
  Cite this work as:                                                                                       
  Manathunga, M.; Chi, J.; Cruzeiro, V.W.D.; Keipert, K.; Pekurovsky, D.; Mu, D.; Miao, Y.;He, X.; Ayers,K;
  Brothers, E.; GÃ¶tz, A.; Merz,K. M. QUICK-21.03 University of California San Diego, CA and Michigan State
  University, East Lansing, MI, 2021                                                                       
                                                                                                           
  If you have any comments or queries, please send us an email for technical support:                      
  quick.merzlab@gmail.com                                                                                  
                                                                                                           
| TASK STARTS ON: Sat Feb 27 20:38:35 2021
| INPUT FILE :    grad_ch3conhch3_b3lyp_ccpvdz.i
| OUTPUT FILE:    grad_ch3conhch3_b3lyp_ccpvdz.o
| DATE FILE  :    grad_ch3conhch3_b3lyp_ccpvdz.d
| BASIS SET PATH: /server-home1/madu/Work/Qk/issue66/basis

 @ Read Job And Atom

  KEYWORD=DFT B3LYP BASIS=CC-PVDZ CUTOFF=1.0E-9 DENSERMS=1.0E-6 GRADIENT DIPOLE                                                                                                                                   
 ============== JOB CARD =============
 METHOD = DENSITY FUNCTIONAL THEORY
 DENSITY FUNCTIONAL = B3LYP
 DIRECT SCF 
 SAD INITAL GUESS 
 USE DIIS SCF
 PRINT ENERGY EVERY CYCLE
 DIPOLE
 STANDARD GRID = SG1
 GRADIENT CALCULATION
 MAX SCF CYCLES =    200
 MAX DIIS CYCLES =   10
 DELTA DENSITY START CYCLE = 1000
 COMPUTATIONAL CUTOFF: 
      TWO-e INTEGRAL   =  0.100E-08
      BASIS SET PRIME  =  0.100E-19
      MATRIX ELEMENTS  =  0.100E-09
      BASIS FUNCTION   =  0.100E-09
      GRADIENT CUTOFF  =  0.100E-08
 DENSITY MATRIX MAXIMUM RMS FOR CONVERGENCE  =  0.100E-05
 BASIS SET = CC-PVDZ,  TYPE = CARTESIAN
| BASIS FILE = /server-home1/madu/Work/Qk/issue66/basis/CC-PVDZ.BAS

 @ Finish reading job


 @ Begin SAD initial guess

 For Atom Kind =    1
 ELEMENT = H 
 BASIS FUNCTIONS =    5
 For Atom Kind =    2
 ELEMENT = C 
 BASIS FUNCTIONS =   15
 For Atom Kind =    3
 ELEMENT = O 
 BASIS FUNCTIONS =   15
 For Atom Kind =    4
 ELEMENT = N 
 BASIS FUNCTIONS =   15

 @ Finish SAD initial guess


 @ Begin Reading Molecular Information


 =========== Molecule Input ==========
 TOTAL MOLECULAR CHARGE  =    0    MULTIPLICITY                =    1
 TOTAL ATOM NUMBER       =   12    NUMBER OF ATOM TYPES        =    4
 NUMBER OF HYDROGEN ATOM =    7    NUMBER OF NON-HYDROGEN ATOM =    5
 NUMBER OF ELECTRONS     =   40

 -- INPUT GEOMETRY -- :
    H           5.4269       5.8668       1.9440
    C           5.2779       6.1654       0.8518
    H           5.9666       6.8917       0.4522
    H           4.4105       6.6466       0.9547
    C           5.1170       4.9327      -0.0230
    O           4.1460       4.6758      -0.6787
    N           6.1595       4.0934       0.1359
    H           6.7649       4.3215       0.9157
    C           6.2481       2.8038      -0.5368
    H           5.5175       2.6865      -1.3191
    H           7.1868       2.6685      -1.0047
    H           6.1320       2.0089       0.2504

 -- DISTANCE MATRIX -- :
              1           2           3           4           5
      1     0.00000
      2     2.15824     0.00000
      3     3.56911     2.03664     0.00000
      4     3.05881     1.88456     3.12459     0.00000
      5     4.15656     2.87257     4.13394     3.96057     0.00000
      6     5.95734     4.56769     5.82574     4.86292     2.26664
      7     4.98219     4.46501     5.33420     6.04957     2.54698
      8     4.32398     4.47791     5.16071     6.25346     3.76543
      9     7.60849     7.11337     7.96570     8.52858     4.65791
     10     8.61231     7.76225     8.66455     8.87926     4.95866
     11     8.86791     8.30591     8.75159     9.88680     6.08668
     12     8.07269     8.09885     9.24040     9.44257     5.87144
              6           7           8           9          10
      6     0.00000
      7     4.24960     0.00000
      8     5.83256     1.91463     0.00000
      9     5.32598     2.75370     4.08814     0.00000
     10     4.72370     4.01249     5.73904     2.03485     0.00000
     11     6.91284     3.95791     4.85418     1.99843     3.21001
     12     6.52427     3.94547     4.70216     2.12543     3.43282
             11          12
     11     0.00000
     12     3.33947     0.00000

============== BASIS INFOS ==============
 BASIS FUNCTIONS =  110
 NSHELL =   51 NPRIM  =  256
 JSHELL =   51 JBASIS =  167


 @ End Reading Molecular Information 


 @ Begin XC Quadrature Formation

 OCTAGO: OCTree Algorithm for Grid Operations 
   PRUNING CUTOFF       = 0.100E-09
   INITIAL GRID POINTS  =       45344
|   FINAL GRID POINTS    =       30180
|   SIGNIFICANT NUMBER OF BASIS FUNCTIONS     =       41461
|   SIGNIFICANT NUMBER OF PRIMITIVE FUNCTIONS =       57253

 @ End XC Quadrature Formation


 @ Begin Energy Calculation

                                         SCF ENERGY
------------------------------------------------------------------------------------------------------------------------
NCYC       ENERGY         DELTA_E      SCF_TIME  DII_CYC   DII_TIME   O_TIME  DIAG_TIME    MAX_ERR    RMS_CHG    MAX_CHG
------------------------------------------------------------------------------------------------------------------------
|  1   -249.178391749      ------         3.568     1        0.01      3.56      0.00    0.2246E+01  0.6330E-01  0.9089E+00
|  2   -248.314962789  -.863429E+00       7.104     2        0.01      7.09      0.00    0.1961E+00  0.3449E-01  0.8891E+00
|  3   -248.032940159  -.282023E+00       6.981     3        0.01      6.97      0.00    0.3340E+00  0.2225E-01  0.6583E+00
|  4   -248.486764755  0.453825E+00       7.006     4        0.01      6.99      0.00    0.6701E-01  0.5214E-02  0.1252E+00
|  5   -248.517004585  0.302398E-01       7.308     5        0.01      7.30      0.00    0.1033E-01  0.9124E-03  0.3087E-01
|  6   -248.517718561  0.713976E-03       7.149     6        0.01      7.14      0.00    0.4379E-02  0.3545E-03  0.1166E-01
|  7   -248.517837617  0.119056E-03       6.618     7        0.01      6.61      0.00    0.1273E-02  0.8978E-04  0.2887E-02
|  8   -248.517848202  0.105851E-04       6.472     8        0.01      6.46      0.00    0.1764E-03  0.1941E-04  0.6257E-03
|  9   -248.517848414  0.211351E-06       6.484     9        0.01      6.47      0.00    0.7113E-04  0.7022E-05  0.2180E-03
    | -------------- 2E-INT CUTOFF CHANGE TO 0.1000E-08 ------------
| 10   -248.517849407  0.993700E-06       8.791    10        0.01      8.78      0.00    0.1692E-04  0.2455E-05  0.4186E-04
| 11   -248.517849410  0.245694E-08       8.809     1        0.01      8.80      0.00    0.9640E-05  0.3694E-05  0.6347E-04
| 12   -248.517849411  0.804675E-09       8.638     2        0.01      8.63      0.00    0.4315E-05  0.4384E-06  0.1192E-04
------------------------------------------------------------------------------------------------------------------------
| REACH CONVERGENCE AFTER  12 CYLCES
| MAX ERROR = 0.431529E-05   RMS CHANGE = 0.438441E-06   MAX CHANGE = 0.119199E-04
 -----------------------------------------------
 ALPHA ELECTRON DENSITY    =   19.9998225353
 BETA ELECTRON DENSITY     =   19.9998225353
 ELECTRONIC ENERGY    =  -429.703140519
 CORE_CORE REPULSION  =   181.185291108
 TOTAL ENERGY         =  -248.517849411

 @ End Energy calculation


 ANALYTICAL GRADIENT: 
----------------------------------------
 COORDINATE    XYZ            GRADIENT
----------------------------------------
    1X     5.4269284182    -0.0151808066
    1Y     5.8668006240    -0.0046522115
    1Z     1.9440425695     0.0224376827
    2X     5.2778867585    -0.0428799390
    2Y     6.1653701423     0.0417511592
    2Z     0.8517884529    -0.0235862605
    3X     5.9665764177    -0.0190271590
    3Y     6.8917330789    -0.0048748458
    3Z     0.4522474392    -0.0008398809
    4X     4.4104958089     0.0828958049
    4Y     6.6466050025    -0.0415365656
    4Z     0.9546988092     0.0032638310
    5X     5.1169733891    -0.0100521359
    5Y     4.9326997712    -0.0090616380
    5Z    -0.0230474609    -0.0318014349
    6X     4.1460066407     0.0286722111
    6Y     4.6757961282     0.0007408575
    6Z    -0.6787269954     0.0279635815
    7X     6.1595319153    -0.0133387836
    7Y     4.0934237942     0.0097180162
    7Z     0.1359075814     0.0111887696
    8X     6.7649152469    -0.0060778566
    8Y     4.3215398203     0.0063896138
    8Z     0.9156517424     0.0048041278
    9X     6.2481135000     0.0087929052
    9Y     2.8038165453     0.0063057777
    9Z    -0.5367658953    -0.0416524547
   10X     5.5175452194     0.0168781274
   10Y     2.6865233246    -0.0044861702
   10Z    -1.3190683672     0.0030291469
   11X     7.1867691112    -0.0319628413
   11Y     2.6684762034     0.0020974574
   11Z    -1.0046973137     0.0106242636
   12X     6.1319766035     0.0012882068
   12Y     2.0088923873    -0.0025233067
   12Z     0.2503952907     0.0146883999
----------------------------------------

 @ Begin Charge and Dipole Calculation

    ATOMIC CHARGES
   ATOM          MULLIKEN            LOWDIN
   H              0.0315             0.1929
   C              0.0262            -0.5757
   H              0.0449             0.2082
   H              0.0001             0.2027
   C              0.0813             0.1616
   O             -0.2744            -0.3084
   N             -0.1637            -0.2843
   H              0.0758             0.2747
   C              0.0769            -0.4656
   H              0.0518             0.2090
   H              0.0187             0.1942
   H              0.0308             0.1908
   TOTAL         0.0000             0.0000

    DIPOLE (DEBYE)
      X         Y         Z        TOTAL
    2.9280   -0.2536    1.8073    3.4502

 @ End Charge and Dipole Calculation


 @ Output Timing Information

------------- TIMING ---------------
| INITIAL GUESS TIME  =    12.534824000(  8.45%)
| DFT GRID OPERATIONS =     0.348678000(  0.24%)
| TOTAL SCF TIME      =    84.972741000( 57.29%)
|       TOTAL OP TIME      =    84.840188000( 57.20%)
|             TOTAL 1e TIME      =     0.044560000(  0.03%)
|             TOTAL 2e TIME      =    67.931103000( 45.80%)
|             TOTAL EXC TIME     =    16.818882000( 11.34%)
|       TOTAL DII TIME      =     0.131991000(  0.09%)
|             TOTAL DIAG TIME    =     0.036189000(  0.02%)
| TOTAL GRADIENT TIME      =    50.444778000( 34.01%)
|       TOTAL 1e GRADIENT TIME      =     0.233005000( 0.20%)
|       TOTAL 2e GRADIENT TIME      =    45.245117000(30.50%)
|       TOTAL EXC GRADIENT TIME     =     4.904690000(  3.31%)
| TOTAL TIME          =   148.322476000
------------------------------------
| Job cpu time:  0 days  0 hours  2 minutes 28.3 seconds.

 @ Finish Output Timing Information

| Normal Termination. Task Finished on: Sat Feb 27 20:41:04 2021
